inductiva/utils/format_utils.py

```diff
@@ -139,6 +139,37 @@ def timedelta_formatter(td: datetime.timedelta) -> str:
     return result
 
 
+def short_timedelta_formatter(td: datetime.timedelta) -> str:
+    """Convert timedelta to short human readable string.
+    
+    This is needed because we need beacause when we want to print text and
+    replace (in the notebooks) there is no way to clear the full line. So, we
+    need to fill the line with white spaces and for that the line needs to have
+    a fixed max length. We defined that max length as 73. So, we need to have
+    a short string to fit in that line and give extra space for the rest of the
+    line.
+    Example:
+        Task {self.id} is about to start. 1 d 2 h 3 m 4 s
```

Comment:
Why did we opted for this format? Will this be used for all time metrics?

Reply:
This is to be used on the wait method. I need a duration and that duration will be formatted by this method.
I need the short version to make sure our lines printed are smaller than 78 chars. (something related with cleaning  the current line for notebooks)